dvc/repo/experiments/queue/base.py

```diff
@@ -213,12 +208,17 @@ def _format_entry(
             for queue_entry in self.iter_queued()
         )
         result.extend(
-            _format_entry(queue_entry, result=exp_result)
-            for queue_entry, exp_result in self.iter_done()
+            _format_entry(queue_entry, status="Failed")
+            for queue_entry, _ in self.iter_failed()
+        )
+        result.extend(
+            _format_entry(queue_entry, status="Success")
+            for queue_entry, _ in self.iter_success()
         )
         return result
 
-    def _remove_revs(self, stash_revs: Mapping[str, ExpStashEntry]):
+    @staticmethod
+    def _remove_revs(stash_revs: Mapping[str, ExpStashEntry], stash: ExpStash):
```

Comment:
Since this is no longer really a private method it should be moved out of the queue classes and into `experiments/stash.py` as `ExpStash.remove_revs(self, stash_revs)`